routes/season/get-season.js

```diff
@@ -68,7 +56,39 @@ const handler = async (req, reply) => {
     return;
   }
 
-  reply.send(season);
+  // Show applications for moderators and admins
+  if (authPayload.roles && (authPayload.roles.includes('moderator')
+  || authPayload.roles.includes('admin'))) {
+    season.depopulate('applications.team');
+    reply.send(season);
+    return;
+  }
+
+  // Check if team already applied for the season
+  let alreadyApplied = false;
+  if (season.applications && season.applications
+    .find(application => String(application.team.members).includes(String(authPayload._id)))
```

Comment:
If you need to find if a value exists in an array, use Array.prototype.includes(). Again, it checks each element for equality with the value instead of using a testing function.

Reply:
Done